homeworks/andrii.triasun_andrey-tryasun/homework_5/sum-array.js

```diff
@@ -0,0 +1,25 @@
+// converting a numeric array to an array of sums
+
+const sumArray = (array) => {
+    const resultArray = [];
+    return () => {
+        array.reduce((acc, cur) => {
+            const sum = acc + cur;
+            resultArray.push(sum);
+            return sum;
+        }, 0);
+        return resultArray;
+    };
+};
```

Comment:
works, but please solve it with `map()`, this will reduce half the amount of code for this task

Reply:
Fixed